go.mod

```diff
@@ -28,3 +28,5 @@ require (
 )
 
 replace github.com/keybase/go-keychain => github.com/99designs/go-keychain v0.0.0-20191008050251-8e49817e8af4
+
+replace github.com/cosmos/cosmos-sdk => github.com/CosmWasm/cosmos-sdk v0.38.1-0.20200727124038-6c7f67fa908f
```

Comment:
👍 The version looks a bit odd but `6c7f67fa9` is on top of `cosmos/backport/v0.39.1`

Reply:
Yeah, it is cuz I haven't pushed all tags to the CosmWasm fork. As soon as they merge my PR into cosmos/cosmos-sdk this will look cleaner